### DISCUSSION TITLE
Import Player in world

### DISCUSSION THREAD

**Discussion by cho66x** [naverz/zepeto-studio-global#48] [General]
HOW TO INCLUDE MY ZEPETO CHARACTER IN THE WORLD I CREATED, So that people can follow my account when they visit my world

**Comment by jhslee** [comment]
Hello @anggacho 👋 

Please use ZepetoCharacterCreator.CreateByZepetoId and replace the "[ZEPETO_ID]" field with your id. It will spawn your character into the world, and allow players to click on your (Make sure your testing using QR code, clicking on characters will not work from the editor) 

```
ZepetoCharacterCreator.CreateByZepetoId("[ZEPETO_ID]", new SpawnInfo(), (character : ZepetoCharacter)=>{
   // Handling Character Instance..
})
```

**Comment by cho66x** [comment]
wow this is cool. I will try it. thank you for your answer @jhslee

**Comment by cho66x** [comment]
By the way, where is the file where I put the code?
